packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,657 +1,659 @@
+## 2.4.8
+
+- Updates README "Usage" section
+
 ## 2.4.7
 
-* Updates README via code excerpts.
-* Fixes violations of new analysis option use_named_constants.
+- Updates README via code excerpts.
```

Comment:
It looks like your editor mass-rewrote this file; please revert all of these changes and use `*` for the new section as well.